CONTRIBUTING.md

```diff
@@ -61,3 +93,78 @@ by two units (that is, four characters).
 >>>>let y = ... in
 >>>>...
 ```
+
+If-then-else
+------------
+
+* If we can write the `if-then-else` instruction on one line, you should do it!
+* If it is not possible, follow one of these templates:
+```OCaml
+if (* test *) then
+>> (* first case *)
+else (* second case *)
+```
+or, if it is not possible to write the second case on one line:
+```OCaml
+if (* test *) then
+>> (* first case *)
+else 
+>> (* second case *)
+```
+It is not readable if the `test` statement cannot be written on one line, so you must write `if (* test *) then` on one line only.
+
+Record
+------
+
+Please follow this template:
+```OCaml
+(** Description of the inductive type*)
+type name =
+  { constructor1 : type1  (** Description of the field 1  *)
+         ...
+  ; constructorN : typeN  (** Description of the field N  *) }
+```
+If it is not possible to write the description on one line, the template is:
+```OCaml
+(** Description of the inductive type*)
+type name =
+  { constructor1 : type1
+  (** Description of the field 1  *)
+         ...
+  ; constructorN : typeN
+  (** Description of the field N  *) }
+```
+
+Do the same thing with sum-types.
+
+Let-in construction
+-------------------
+
+The template is very simple:
+```OCaml
+let (* name *) = (* ... *) in
+```
+or, if you need several lines:
+```OCaml
+let (* name *) =
+ (* ... *)
+in
+```
+
+Building of examples
+--------------------
+
+Don't forget to build your own examples about your job.
+Moreover, the naming convention is to start the name of an object with a lowercase letter, and the name of a type with an uppercase letter.
+For example:
```

Comment:
This paragraph is mixing naming conventions and model for pull requests: the two are good ideas, but let's make things clear.

Reply:
I agree so you would like to add sub-sections ? sections ?